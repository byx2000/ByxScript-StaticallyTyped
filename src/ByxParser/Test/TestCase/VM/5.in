int g1 = 100;
int g2 = g1;
int g3 = g2;

double g4 = 3.14;
double g5 = g4;
double g6 = g5;

int g7 = 55;
int g8 = 23.75;
double g9 = 3;

function void change()
{
	g7 = 45;
}

function void main()
{
	change();
}