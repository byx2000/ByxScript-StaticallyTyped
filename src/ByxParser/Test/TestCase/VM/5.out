i 100
i 100
i 100
d 3.14
d 3.14
d 3.14
i 45
i 23
d 3